src/main/java/org/inferred/freebuilder/processor/Analyser.java

```diff
@@ -70,6 +57,19 @@
 import javax.lang.model.util.SimpleTypeVisitor6;
 import javax.lang.model.util.Types;
 
+import org.inferred.freebuilder.processor.Metadata.Property;
```

Comment:
Because non-Google Eclipses aren't crazy, and I'm not going to tweak the output by hand :)